inc/features/blocks/personalization-variant/edit.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+const { InnerBlocks } = wp.blockEditor;
+const { compose } = wp.compose;
+const { withSelect, withDispatch } = wp.data;
+
+const Edit = ( {
+	hasChildBlocks,
+	isSelected,
+	selectParent,
+} ) => {
```

Comment:
IMO, this would look a bit nicer if it wasn't unpacking; we're not using these variables repeatedly, so writing `props.hasChildBlocks` isn't a huge amount of work.

Reply:
Left this as is seeing as we're now creating a new `props` var to pass through to the `InnerBlocks`.